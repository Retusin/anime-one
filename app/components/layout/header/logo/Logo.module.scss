.logo {
  @apply font-bold text-[26px] cursor-pointer;
}
