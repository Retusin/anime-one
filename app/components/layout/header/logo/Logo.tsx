import Link from 'next/link';
import { FC } from 'react';

import styles from './Logo.module.scss';

export const Logo: FC = () => (
  <Link href='/'>
    <div className={styles.logo}>AnimeOne</div>
  </Link>
);
