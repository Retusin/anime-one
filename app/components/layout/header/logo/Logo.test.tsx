import { Logo } from '@/components/layout/header/logo/Logo';
import { render } from '@testing-library/react';

test('render logo', () => {
  const { asFragment } = render(<Logo />);
  // @ts-ignore
  expect(asFragment(<Logo />)).toMatchSnapshot();
});
