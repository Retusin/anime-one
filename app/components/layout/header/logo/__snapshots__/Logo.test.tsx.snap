// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render logo 1`] = `
<DocumentFragment>
  <div
    class="logo"
  >
    AnimeOne
  </div>
</DocumentFragment>
`;
